_posts/2022-04-10-using-log-collection.markdown

```diff
@@ -0,0 +1,156 @@
+---
+layout: post
+title:  "Log Collection on Flotta devices"
+date:   2022-04-10 12:00:00 +0100
+categories: flotta
+author: Eloy Coto <eloycoto@acalustra.com>
+tags:
+  - guide
+  - flotta
+  - logs
+
+summary-1: Introduction to flotta log collection
+---
+
+When running thousands of devices on the far edge, log collection retrieval
+maybe differs from one site to another, as also the limit of data that can be
+sent from the device or stored in the device.
+
+At the same time, workloads logs can vary in terms of importance, so the backup
+workload logs maybe need to be always pushed, and maybe another workload with
+less importance can lose logs on network connectivity issues. At the same time,
+maybe you don’t want that workload that manages any personal information sends
+the information outside the country.
+
+For that, Flotta introduces log collection targets, so each device - or
+deviceSets- will have a set of custom logs servers that one workload can use to
+send the logs.
+
+![Device Logs diagram](./assets/images/DeviceLogsConfig.png)
+
+So the main use case is to be able to define servers in edge devices and be
+consumed by the workloads, so in this way, the roles will be limited between
+developers and edge administrators.
+
+*A sample workflow for a retail store can be:*
+
+**Edge Device Administrator:** defines the device security, heartbeat and
+general options based on the device or deviceSet. In this case, will define two
+kinds of logs servers, LogSecure which always push the logs to the server, with
+big storing buffers. Meanwhile, there is a logGeneral, that it’s the server
```

Comment:
Meanwhile ->  At the same time?
"that it’s the server" -> "that is the server that may lose logs on network downtime, or maybe using different transports"